providers/fips/self_test_kats.c

```diff
@@ -142,39 +144,78 @@ static int self_test_cipher(const ST_KAT_CIPHER *t, OSSL_SELF_TEST *st,
     return ret;
 }
 
+static int add_params(OSSL_PARAM_BLD *bld, const ST_KAT_PARAM *params,
+                      BN_CTX *ctx)
+{
+    int ret = 0;
+    const ST_KAT_PARAM *p;
+
+    if (params == NULL)
+        return 1;
+    for (p = params; p->data != NULL; ++p)
+    {
+        switch (p->type) {
+        case OSSL_PARAM_UNSIGNED_INTEGER: {
+            BIGNUM *bn = BN_CTX_get(ctx); /* grab a bn from the pool */
```

Comment:
This comment feels a little bit superfluous.

Reply:
removed